user-service/src/main/java/rs/edu/raf/banka1/services/CardServiceImlp.java

```diff
@@ -0,0 +1,104 @@
+package rs.edu.raf.banka1.services;
+
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import rs.edu.raf.banka1.model.BankAccount;
+import rs.edu.raf.banka1.model.Card;
+import rs.edu.raf.banka1.model.Customer;
+import rs.edu.raf.banka1.model.User;
+import rs.edu.raf.banka1.repositories.CardRepository;
+import rs.edu.raf.banka1.repositories.CustomerRepository;
+import rs.edu.raf.banka1.repositories.UserRepository;
+
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@Getter
+@Setter
+@Service
+public class CardServiceImlp implements CardService{
+    @Autowired
+    private CardRepository cardRepository;
+    @Autowired
+    private CustomerRepository customerRepository;
+
+
+    @Override
+    public List<Card> getAllCardsByAccountNumber(String accountNumber) {
+        return cardRepository.findByAccountNumber(accountNumber);
+    }
+
+    @Override
+    public Card createCard(String cardType, String cardName, String accountNumber, Integer limit) {
```

Comment:
zasto ovo ne cuva karticu u bazu?

Reply:
Funkcionalno programiranje! Imam posebnu save funkciju koja prilikom čuvanja napravi i sačuva 2 kartice, tako sam se dogovorio sa Ivanom, jer je front rekao da hard-kodujemo to.